paparazzi-gradle-plugin/src/test/projects/build-class-next-sdk/gradle.properties

```diff
@@ -1 +1,2 @@
 android.suppressUnsupportedCompileSdk=UpsideDownCakePrivacySandbox
+android.useAndroidX=true
```

Comment:
is this necessary?  if so, how come?

Reply:
Got following error without android.useAndroidX=true. Previously `gradle.properties` was programmatically created, so `android.useAndroidX=true` was dynamically added

```
* What went wrong:
Execution failed for task ':generateDebugUnitTestStubRFile'.
> Configuration `:debugUnitTestRuntimeClasspath` contains AndroidX dependencies, but the `android.useAndroidX` property is not enabled, which may cause runtime issues.
  Set `android.useAndroidX=true` in the `gradle.properties` file and retry.
  The following AndroidX dependencies are detected:
  :debugUnitTestRuntimeClasspath -> app.cash.paparazzi:paparazzi:1.3.2-SNAPSHOT:20230726.225900-20 -> androidx.annotation:annotation:1.6.0
  :debugUnitTestRuntimeClasspath -> app.cash.paparazzi:paparazzi:1.3.2-SNAPSHOT:20230726.225900-20 -> androidx.annotation:annotation:1.6.0 -> androidx.annotation:annotation-jvm:1.6.0
```